app-tests/gromacs/gromacs_simple.py

```diff
@@ -0,0 +1,138 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+from hpctestlib.sciapps.gromacs.benchmarks import gromacs_check
+
+
+@rfm.simple_test
+class simple_gromacs_check(gromacs_check):
+    maintainers = ['mszpindler']
+    use_multithreading = False
+    executable_opts += ['-dlb yes', '-ntomp 1', '-npme -1']
+    pe_release = '21.12'
+    gromacs_version = '2021.5'
+    num_nodes = 1
+
+    benchmark_info = parameter([
+        ('benchMEM', -1.08735e+06, 0.001),
+        ('benchPEP', -1.08735e+06, 0.001), # Fix it
+    ], fmt=lambda x: x[0], loggable=True)
+
+    allref = {
+	1: {
+            'cascadelake': {  # Copied from Cascade Lake results 
+                'benchMEM': (92.043, None, None, 'ns/day'),
+                'benchPEP': (0.561, None, None, 'ns/day'), 
+            }
+        }
+    }
+
+    @run_after('init')
+    def setup_modules(self):
+        if self.current_system.name in ('lumi'):
+            if self.nb_impl == 'cpu':
+                self.modules = ['GROMACS-{gromacs_version}-cpeGNU-{pe_release}-CPU']
+                self.valid_prog_environs = ['cpeGNU']
+            # Add module information for GPU enabled version    
+            #elif self.nb_impl == 'gpu':
+            #    self.modules = ['GROMACS-{gromacs_version}-cpeGNU-{pe_release}-GPU']
+            #    self.valid_prog_environs = ['cpeGNU']
+
+    @run_after('init')
+    def prepare_test(self):
+        self.__bench, self.__nrg_ref, self.__nrg_tol = self.benchmark_info
+        self.descr = f'GROMACS {self.__bench} benchmark (NB: {self.nb_impl})'
+        self.prerun_cmds = [
+            f'curl -LJO https://www.mpinat.mpg.de/{self.__bench} && unzip {self.__bench}.zip && ln -s {self.__bench}.tpr benchmark.tpr'
+        ]
+        self.executable_opts += ['-nb', self.nb_impl, '-s benchmark.tpr']
+
+    @loggable
+    @property
+    def bench_name(self):
+        '''The benchmark name.
+
+        :type: :class:`str`
+        '''
+
+        return self.__bench
+
+    @property
+    def energy_ref(self):
+        '''The energy reference value for this benchmark.
+
+        :type: :class:`str`
+        '''
+        return self.__nrg_ref
+
+    @property
+    def energy_tol(self):
+        '''The energy tolerance value for this benchmark.
+
+        :type: :class:`str`
+        '''
+        return self.__nrg_tol
+
+    @run_after('init')
+    def setup_nb(self):
+        valid_systems = {
+            'cpu': {
+                1: ['lumi:cpu'],
```

Comment:
Is there a `lumi:cpu` partition on the config?

Reply:
No, there is not; fixed with 6181b10